config/horizon.php

```diff
@@ -165,36 +165,36 @@
                 'connection' => 'redis',
                 'queue' => ['bpmn'],
                 'balance' => 'false',
-                'processes' => 8,
-                'tries' => 1,
-                'timeout' => 3600,
+                'processes' => env('PM4_HORIZON_SUPERVISOR_BPMN_PROCESSES', 1),
+                'tries' => env('PM4_HORIZON_SUPERVISOR_BPMN_TRIES', 1),
```

Comment:
At this moment, this parameter `tries` (in bpmn queue) must always be always `1`, since retry multiple times a BPMN Action could cause a token to be injected to the next task in every retry.